### PR TITLE
Update Milestones site functionality

### PR DESCRIPTION
**Pull request by Max1us**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: locales/en/faq.json [hunk 1]
```diff
@@ -94,9 +94,9 @@
       "question": "Where Can I Go to Learn More About Verge?",
       "answer": "We are on a ton of different social platforms and are very active on Twitter, Telegram, Slack, and Discord."
     },
-    "roadmap": {
-      "question": "Where Can I Find the Verge Roadmap?",
-      "answer": "We have a <a href='/roadmap' target='_blank'>roadmap</a> listed on our site that usually lists items we are working on for the next month or two. We don't like to put long-term items (greater than 3 months) on our roadmap, because things change too fast in crypto to anticipate that far ahead."
+    "milestones": {
```
Comment (marpme) [inline locales/en/faq.json:97]: Then you need to adjust POEditor keys as well. I would just leave the key as roadmap

File: lists/vendors.json [hunk 2]
```diff
@@ -1263,20 +1263,6 @@
...
-            "url": "https://nomadpostbox.com/",
-            "categories": {
-                "retail": false,
-                "foodAndBev": false,
-                "service": true,
-                "entertainment": false,
-                "travel": false,
-                "education": false
-            }
-        },
```
Comment (marpme) [inline lists/vendors.json:1279]: the vendor should stay in

File: locales/en/faq.json (hunk 1 above)
Comment (Max1us) [reply locales/en/faq.json:97]: Yep, they're in there twice

File: lists/vendors.json (hunk 2 above)
Comment (Max1us) [reply lists/vendors.json:1279]: Yep, they're in there twice

**Pull request merged**